Adding the developer community CoC

joannasmith opened the pull request.

joannasmith committed:
Adding the developer community CoC

anthonyroux commented on Developer Community Code of Conduct.md:
```diff
@@ -0,0 +1,45 @@
+
+This isn’t an exhaustive list of things that you can’t do. Rather, take it in the spirit in which it’s intended – a guide to make it easier to enrich all of us and the technical communities in which we participate.
+
+This code of conduct outlines our expectations for all those who participate in our Miro developer community, as well as the consequences for unacceptable behavior. Your participation here is also subject to the [Miro Community Guidelines](https://community.miro.com/community-welcome-guide-14/miro-community-guidelines-28) and the [Miro Terms of Use](https://community.miro.com/site/terms).
```
Do we want to call it `Miro developer community` or `Miro platform community` to be more inclusive with people using the platform but that are not necessarily developers?

joannasmith replied:
Good point. Updated!

anthonyroux reviewed (approved):
Looks very good, great job Joanna! I added only a very minor feedback, let me know what you think about it

joannasmith committed:
Update to "platform community"

Replacing the "developer community" term with the more-applicable "platform community".
